key-wallet-ffi/include/key_wallet_ffi.h

```diff
@@ -92,6 +92,10 @@ typedef enum {
     PROVIDER_PLATFORM_KEYS = 10,
     DASHPAY_RECEIVING_FUNDS = 11,
     DASHPAY_EXTERNAL_ACCOUNT = 12,
+    /*
+     Platform Payment address (DIP17/DIP18) - Path: m/9'/5'/17'/account'/key_class'/index
```

Comment:
```suggestion
     Platform Payment address (DIP17) - Path: m/9'/5'/17'/account'/key_class'/index
```

Should this still mention DIP18? Just wonder since you removed it from the PR title also. Its in a few other places as well.